### PR TITLE
update triage docs: needs-triage, p3 crashes/freezes

### PR DESCRIPTION
**Pull request by seanbudd**
Adds information about the "needs-triage" label.

Adds a classification for P3s: a crash/freeze that affects only one user

### CODE REVIEW

File: projectDocs/issues/triage.md [hunk 1]
```diff
@@ -140,15 +140,22 @@ We also have a label for `enhancement`, think of this as a more internal facing
...
 A `triaged` issue that requires a complex fix may require advice from NV Access, such as a project plan, before implementation is started.
-An issue with a simple solution should get labelled `good first issue`.
 
-For controversial changes, a product decision from NV Access may be required before applying the `triaged` label. This can be indicated with adding the label `blocked/needs-product-decision`.
+An issue with a simple solution should get labelled `good first issue`.
+If it is a complex issue, technical investigation may be required.
+This can be indicated with adding the label `blocked/needs-technical-investigation`.
 
 ### Priority
 Bugs/regressions are given priorities based on an estimate of their severity and impact.
```
Comment (gerald-hartig) [inline projectDocs/issues/triage.md:161]: Is it worth mentioning in L164 that P1s can also be a patch release? Are they considered milestones?

**Comment by gerald-hartig** [comment]
I like the improvements. Nice work.

**Review by gerald-hartig — APPROVED** [review]

File: projectDocs/issues/triage.md (hunk 1 above)
Comment (seanbudd) [reply projectDocs/issues/triage.md:161]: I don't think it's worthwhile documenting this. Sometimes we create milestones for patch releases, particularly when there are multiple issues involved

**Review by Qchristensen — APPROVED** [review]
Reads well, good work.

**Pull request merged**